src/main/java/bio/terra/cli/command/Main.java

```diff
@@ -104,17 +108,29 @@ public static int runCommand(String... args) {
    * @param args from stdin
    */
   public static void main(String... args) {
+    argList = Arrays.asList(args);
```

Comment:
nit: please add a comment here explaining why we need to save the args in this class given that we're logging them in `BaseCommand`